maltd-frontend/cypress/integration/tests/add-project.js

```diff
@@ -0,0 +1,23 @@
+/* global cy, before */
+// / <reference types="Cypress" />
+
+import LandingPage from "../../support/pageObjects/landing-page";
+// import UserPage from "../../support/pageObjects/user-page";
```

Comment:
do we still need this comment?

Reply:
Yes. This page will be uncommented when extending tests in the next commits.